iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/Program.cs

```diff
@@ -15,8 +15,8 @@ namespace Microsoft.Azure.Devices.Client.Samples
 {
     public class Program
     {
-        // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json
-        private const string ModelId = "dtmi:com:example:TemperatureController;1";
+        // DTDL interface used: https://github.com/Azure/iot-plugandplay-models
```

Comment:
nit: maybe point to the actual file https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/com/example/temperaturecontroller-2.json

Reply:
what I did in my second commit, do you see it?

Reply:
done